src/applications/mhv-secure-messaging/tests/containers/App.unit.spec.jsx

```diff
@@ -17,6 +17,7 @@ describe('App', () => {
     delete global.window.location;
     global.window.location = {
       replace: sinon.spy(),
+      pathname: '/my-health/secure-messages',

```

Comment:
Note:Sec Nav requires a `window.location.pathname`

Reply:
Does it make sense to use prop-types and/or default function arguments to enforce this requirement of the `<MhvSecondaryNav />`?

Reply:
Well, it is not a property you pass, but window.location.pathname is used in the code.

Reply:
I changed the Sec Nav code to handle a missing pathname. This will help testing other apps using the Sec Nav.